packages/commonwealth/client/scripts/views/components/profile/profile_header.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import jdenticon from 'jdenticon';
+
+import 'components/profile/profile_header.scss';
+
+import app from 'state';
+import type { NewProfile as Profile } from 'client/scripts/models';
+import { CWButton } from '../component_kit/cw_button';
+import { CWText } from '../component_kit/cw_text';
+import { renderQuillTextBody } from '../quill/helpers';
+import { SocialAccounts } from '../social_accounts';
+
+type ProfileHeaderProps = {
+  profile: Profile;
+  isOwner: boolean;
+};
+
+const ProfileHeader = (props: ProfileHeaderProps) => {
+  const navigate = useNavigate();
+  const [defaultAvatar, setDefaultAvatar] = React.useState<string>();
+
+  React.useEffect(() => {
+    setDefaultAvatar(jdenticon.toSvg(props.profile.id, 90));
+  }, []);
```

Comment:
why using useEffect when this can be initialized right away in defaultAvatar. But anyways why do we need state here if its not set anywhere. It can be just a const.